UPGRADE-4.0.md

```diff
@@ -24,6 +24,9 @@ Serializer
 Yaml
 ----
 
+ * The `Dumper::setIndentation()` method was removed. Pass the indentation
+   level to the constructor instead.
```

Comment:
well, forgetting to remove deprecated APIs would be a failure in our process anyway